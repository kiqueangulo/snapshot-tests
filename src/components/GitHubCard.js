import Card from "react-bootstrap/Card";

function GitHubCard() {
    return (
        <Card style={{ width: '18rem' }}>
        <Card.Img style={{ height: '18rem'}} variant="top" src='https://scontent-atl3-2.xx.fbcdn.net/v/t1.6435-9/39811523_10216104584714572_3458017168036200448_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=0debeb&_nc_ohc=JHwsCvPcGioAX_xxd5d&tn=5ClC7_3Dd0oRpZ0J&_nc_ht=scontent-atl3-2.xx&oh=00_AT_eR-zCUuVKTOJ3Gl2_SSj7Im159erLLFoiGKp9JGWlpw&oe=62E21F0F' />
            <Card.Body>
                <Card.Title>kiqueangulo</Card.Title>
                <Card.Text>
                    Living the life.
                </Card.Text>
            </Card.Body>
            <Card.Body>
                <Card.Link href="https://github.com/kiqueangulo">GitHub</Card.Link>
            </Card.Body>
        </Card>
    );
}

export default GitHubCard;