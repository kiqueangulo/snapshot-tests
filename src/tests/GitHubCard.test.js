import renderer from 'react-test-renderer';

import GitHubCard from '../components/GitHubCard';

test('renders my card', () => {
    const tree = renderer.create(<GitHubCard />).toJSON();
    expect(tree).toMatchSnapshot();
});