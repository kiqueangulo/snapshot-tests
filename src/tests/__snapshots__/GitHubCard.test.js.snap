// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders my card 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="https://scontent-atl3-2.xx.fbcdn.net/v/t1.6435-9/39811523_10216104584714572_3458017168036200448_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=0debeb&_nc_ohc=JHwsCvPcGioAX_xxd5d&tn=5ClC7_3Dd0oRpZ0J&_nc_ht=scontent-atl3-2.xx&oh=00_AT_eR-zCUuVKTOJ3Gl2_SSj7Im159erLLFoiGKp9JGWlpw&oe=62E21F0F"
    style={
      Object {
        "height": "18rem",
      }
    }
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      kiqueangulo
    </div>
    <p
      className="card-text"
    >
      Living the life.
    </p>
  </div>
  <div
    className="card-body"
  >
    <a
      className="card-link"
      href="https://github.com/kiqueangulo"
    >
      GitHub
    </a>
  </div>
</div>
`;
